apps/src/code-studio/components/SoundLibrary.jsx

```diff
@@ -73,6 +74,10 @@ export default class SoundLibrary extends React.Component {
     selectedSound: {}
   };
 
+  componentWillMount() {
+    this.sounds = new Sounds();
```

Comment:
`Sounds` is designed as a singleton, because some browsers can't handle more than one `AudioContext` being created in a window/tab.  Please use [`Sounds.getSingleton`](https://github.com/code-dot-org/code-dot-org/blob/9f3c57bb19125e1c60e07e886a661c6163b75be7/apps/src/Sounds.js#L81-L86) instead.  (it actually looks like `SoundList` [needs to do this too](https://github.com/code-dot-org/code-dot-org/blob/9f3c57bb19125e1c60e07e886a661c6163b75be7/apps/src/code-studio/components/SoundList.jsx#L27)).

It can be a little hard to track down this history of our `Sounds` class, because it's moved around a few times.  [A lot of that history lives here](https://github.com/code-dot-org/code-dot-org/commits/9257d2b073f811119b31d665f23a620ba3d60dd2/dashboard/app/assets/javascripts/soundEffects.js), and [this PR](https://github.com/code-dot-org/code-dot-org/pull/5462) contains a lot of details about the last time we made major changes to our approach to web audio.